src/api/README.md

```diff
@@ -1648,6 +1652,119 @@ Result:
 }
 ```
 
+## Filesystem APIs
+
+### Save data
+
+API sets: `FILESYSTEM`
+
+```
+URI: /api/v2/data
+Method: POST
+Args:
+    data: arbitrary data to save [required]
+    update: update existing values [optional]
+```
+
+Saves arbitrary data to disk.
+
+If `update` is set to true then old values are updated,
+if a key does not exist then its added.
+
+Example:
+
+```sh
+curl -X POST \
+  http://127.0.0.1:6420/api/v2/data \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"data": {"key1": "value1", "key2": "value2", "key3": "value3", "key4": "value4"}
+}'
+```
+
+Result:
+
+```json
+"success"
+```
+
+Example(update):
+
+```sh
+curl -X POST \
+  http://127.0.0.1:6420/api/v2/data \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"data": {"key2": "value4", "key4": "value6"},
+	"update": true
+}'
+```
+
+Result:
+
+```json
+"success"
+```
+
+
+### Get data
+
+API sets: `FILESYSTEM`
+
+```
+URI: /api/v2/data
+Method: GET
+Args:
+    keys: list of keys to retrieve [required]
+```
+
+Get data from data file on disk
+
+Example:
+
+```sh
+curl -X GET \
```

Comment:
Isn't `-F` for formdata? Should be a query string

Reply:
both work apparently. I been using form data only in postman